manifests/params.pp

```diff
@@ -60,6 +60,12 @@
 # [*rhn_password*]
 # The Redhat Network password. Must be set if the param rhn is true.
 #
+# [*rhnget_cleanup*]
+# Clean up packages that are not on the sending side? mrepo default yes.
```

Comment:
But the default in this class is now `false` (translated to `no` in template)?  Can we change this to be backwards compatible?

One option might be to default the new parameters to `undef` instead of `true`/`false`.  Then in the template don't write any `rhnget_cleanup` line unless the module user has explicitly overridden with `true`/`false`.